python/ray/autoscaler/_private/_kubernetes/node_provider.py

```diff
@@ -10,14 +11,22 @@
 from ray.autoscaler._private._kubernetes.config import bootstrap_kubernetes, \
     fillout_resources_kubernetes
 from ray.autoscaler.node_provider import NodeProvider
+from ray.autoscaler.tags import NODE_KIND_HEAD
 from ray.autoscaler.tags import TAG_RAY_CLUSTER_NAME
+from ray.autoscaler.tags import TAG_RAY_NODE_KIND
 
 logger = logging.getLogger(__name__)
 
 MAX_TAG_RETRIES = 3
 DELAY_BEFORE_TAG_RETRY = .5
 
 
+def head_service_selector(cluster_name: str) -> Dict[str, str]:
+    """Selector for Operator-configured head service.
+    """
+    return {"cluster.ray.io/component": f"{cluster_name}-ray-head"}
```

Comment:
should `cluster.ray.io/component` be a constant?

Reply:
Yes.